src/clients/python/experimental_api_v2/library/CMakeLists.txt

```diff
@@ -38,12 +38,14 @@ if(${TRTIS_ENABLE_HTTP_V2} OR ${TRTIS_ENABLE_GRPC_V2})
     target_link_libraries(cshmv2 rt)
 
     #
-    # libccudashm.so
+    # libccudashmv2.so
```

Comment:
Is it necessary to have v2 of the utility libraries? I think both v1 and v2 clients can use it?

Reply:
We have renamed every 'trtis' occurence to 'triton' in the v2 libraries.